app/agency/api/views.py

```diff
@@ -116,9 +120,31 @@ def get_custom_request_form_fields():
             field_required = value['required']
             min_length = value.get('min_length', None)
             max_length = value.get('max_length', None)
+            character_counter = value.get('character_counter', None)
+            placeholder = value.get('placeholder', None)
+            popover = value.get('popover', None)
+            tooltip = value.get('tooltip', None)
+
+            if max_length and character_counter:
```

Comment:
Should we also check if min_length? If so, we need to modify main.js to also take in the min_length value so that the validation appears properly.

Right now if the text is < min_length, the character counter is not shaded red.